### ISSUE TITLE
0.5.0 exits and won't restart...

### ISSUE THREAD

**Issue by mcsherrylabs** [txpipe/dolos#115]
Here is the log from the container.
Restarting the docker results in another exit... 

```
2023-10-16T09:11:21.304574Z  WARN stage{stage="reducer"}:execute:apply_block: dolos::sync::apply: skipping missing utxo
2023-10-16T09:11:21.304920Z  INFO stage{stage="reducer"}:execute:apply_block: dolos::sync::apply: applied block slot=23003774 hash=Hash<32>("7105cb0f1936464a3e2cc7dfea06b2a34b16c2bf866d6214d98eacf2e7834ca5")
2023-10-16T09:11:21.305269Z ERROR stage{stage="reducer"}:execute:apply_block: gasket::framework: x=Invalid CBOR structure: 9223372036854775809 overflows target type at position 1065: when converting u64 to i64
2023-10-16T09:11:21.305370Z ERROR stage{stage="reducer"}: gasket::runtime: stage should stop retry=Retry(None)
2023-10-16T09:11:21.305467Z  INFO stage{stage="reducer"}: gasket::runtime: switching stage phase prev_phase=Working next_phase=Teardown
2023-10-16T09:11:21.305543Z  INFO stage{stage="reducer"}: gasket::runtime: stage teardown ok
2023-10-16T09:11:21.306934Z  INFO stage{stage="reducer"}: gasket::runtime: switching stage phase prev_phase=Teardown next_phase=Ended
2023-10-16T09:11:21.319937Z ERROR stage{stage="reducer"}:execute: gasket::framework: x=error sending work unit through output port
2023-10-16T09:11:21.319984Z ERROR stage{stage="reducer"}: gasket::runtime: stage should stop retry=Retry(None)
2023-10-16T09:11:21.319994Z  INFO stage{stage="reducer"}: gasket::runtime: switching stage phase prev_phase=Working next_phase=Teardown
2023-10-16T09:11:21.320009Z  INFO stage{stage="reducer"}: gasket::runtime: stage teardown ok
2023-10-16T09:11:21.320014Z  INFO stage{stage="reducer"}: gasket::runtime: switching stage phase prev_phase=Teardown next_phase=Ended
2023-10-16T09:11:21.388509Z  INFO stage{stage="chainsync"}:execute: pallas_network::miniprotocols::blockfetch::client: batch start
2023-10-16T09:11:21.388543Z ERROR stage{stage="chainsync"}:execute: gasket::framework: x=error sending work unit through output port
2023-10-16T09:11:21.388558Z ERROR stage{stage="chainsync"}: gasket::runtime: stage should stop retry=Retry(None)
2023-10-16T09:11:21.388566Z  INFO stage{stage="chainsync"}: gasket::runtime: switching stage phase prev_phase=Working next_phase=Teardown
2023-10-16T09:11:21.388603Z  INFO stage{stage="chainsync"}: gasket::runtime: stage teardown ok
2023-10-16T09:11:21.388609Z  INFO stage{stage="chainsync"}: gasket::runtime: switching stage phase prev_phase=Teardown next_phase=Ended
thread 'main' panicked at 'stage stops: TetherDropped', /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/gasket-0.5.0/src/daemon.rs:23:36
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
2023-10-16T09:11:22.444140Z  INFO gasket::daemon: dismissing stage stage="chainsync" state=Dropped

```

**Comment by mcsherrylabs** [comment]
Using the following configuration on an Ubuntu EC2 (small) instance with 300Gb SSD. 

```
[upstream]
peer_address = "preprod-node.world.dev.cardano.org:30002"
network_magic = 2

[rolldb]
path = "./tmp/rolldb"
k_param = 1000

[applydb]
path = "./tmp/applydb"

[serve.grpc]
listen_address = "[::]:50051"

[serve.ouroboros]
listen_address = "localhost:30013"
magic = 0
```

**Comment by mcsherrylabs** [comment]
Fixed by 0.0.6
